cypress/integration/spec.ts

```diff
@@ -1,6 +1,13 @@
 describe('Visit page', () => {
+  before(() => {
+    cy.setupAuth();
+    cy.intercept('GET', 'appuio-api/apis/appuio.io/v1/zones', {fixture: 'zones.json'});
+  });
   it('Visits the initial project page', () => {
```

Comment:
Please also add a test case for when there is no zone and when the call to zones returns with an 403.

Reply:
Is there also a test case that displays a generic error if the API is generally unavailable? (e.g. being restarted or incidents)

Reply:
I added three test cases:
* list with two entries
* empty list
* request failed (every failing request, 403 or anything else)